pkg/account/service.go

```diff
@@ -287,7 +287,6 @@ func (a *Service) Reset(data *Account) error {
 // UpsertPrincipalAccess merges principal access to make sure its in sync with expectations
 func (a *Service) UpsertPrincipalAccess(data *Account) error {
 	err := validation.ValidateStruct(data,
-		validation.Field(&data.Status, validation.NotNil, validation.By(isAccountNotLeased)),
 		validation.Field(&data.AdminRoleArn, validation.NotNil),
```

Comment:
Do you still want to validate Status for not nil?

Reply:
I did a little bit of searching I don't see any code in the account manager that uses Status.  So I don't think we need it at all... but double check my work.

Reply:
Since Update principal policy lambda is triggered when new lease is added, do we need to validate Status to ensure it's not nil and it's of Status "Leased"?

Reply:
Its also triggered on account creation.  In that case the account is NotReady.

Reply:
I'd say this code should only validate for fields that it expects to be in a certain state, in order to do it's job.
So just the role arns, in this case. This change is 👍 to me